basetest.pm

```diff
@@ -17,6 +17,15 @@ use Mojo::File 'path';
 my $serial_file_pos = 0;
 my $autoinst_log_pos = 0;
 
+my $fail_severity = {
+    '' => 0,
+    na => 0,
+    ok => 100,
+    softfail => 200,
+    fail => 300,
+    canceled => 800
```

Comment:
should be "cancelled" :)

Reply:
For os-autoinst it should actually be "canceled" when I remember correctly.

Reply:
Oh, I didn't realize that the typo is already in the existing code...

Reply:
"canceled" is AE and "cancelled" BE. So not really a typo either. Just inconsistent spelling.